crates/rooch-framework/sources/crypto/ecdsa_k1.move

```diff
@@ -2,9 +2,9 @@ module rooch_framework::ecdsa_k1 {
     use std::vector;
 
     /// constant codes
-    const VALID_ECDSA_K1_TO_SCHEME_BITCOIN_LENGTH: u64 = 1;
-    const VALID_ECDSA_K1_COMPRESSED_PUBKEY_LENGTH: u64 = 33;
-    const VALID_ECDSA_K1_SIG_LENGTH: u64 = 64;
+    const ECDSA_K1_TO_SCHEME_BITCOIN_LENGTH: u64 = 1;
```

Comment:
So, the convention for final constants is to choose without any prefix?

Reply:
error code with `Error` prefix

Reply:
I mean CONST prefix

Reply:
Do not need the `CONST` prefix for other `const`.